src/nvim/strings.c

```diff
@@ -588,18 +589,30 @@ static varnumber_T tv_nr(typval_T *tvs, int *idxp)
 ///                  value.
 /// @param[in,out]  idxp  Index in a list. Will be incremented.
 ///
+/// @param[out]  tofree If "tofree" is NULL get_tv_string_chk() is used. Some
```

Comment:
Should be two spaces after the name of the variable. Also modify documentation, it may not be NULL in the code.